__tests__/app.test.js

```diff
@@ -13,3 +25,26 @@ describe("GET /api", () => {
       });
   });
 });
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of topics", () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then((response) => {
+      const topics = response.body.topics
+        expect(topics.length).toBe(3);
+          topics.forEach((topic) => {
+          expect(typeof topic.description).toBe("string");
+          expect(typeof topic.slug).toBe("string");;
+      });
+  });
+});
+test("404: endpoint not available", () => {
+  return request(app)
+    .get('/api/snails')
```

Comment:
is this test related to /api/topics in any way? If not it could get its own describe block! :)